rune/libenclave/attestation/sgx/challenger/challenger-epid.go

```diff
@@ -1,8 +1,9 @@
-package sgx_challenger // import "github.com/inclavare-containers/rune/libenclave/attestation/sgx/sgx_challenger"
+package sgx_challenger // import "github.com/inclavare-containers/rune/libenclave/attestation/internal/sgx/sgx_challenger"
```

Comment:
The import path should be "sgx/challenger"

Reply:
> The import path should be "sgx/challenger"

Done, thanks.